campari/AllASPFuncs.py

```diff
@@ -398,33 +397,36 @@ def findAllExposures(ra, dec, transient_start, transient_end, band, maxbg=None,
         raise RuntimeError(f"Got status code {result.status_code}\n"
                            "{result.text}")
 
-    res = pd.DataFrame(result.json())[["filter", "pointing", "sca", "mjd"]]
-    res.rename(columns={"mjd": "date", "pointing": "Pointing", "sca": "SCA"},
+    res = pd.DataFrame(result.json())[["pointing", "sca", "mjd"]]
+    res.rename(columns={"mjd": "date", "pointing": "pointing", "sca": "sca"},
```

Comment:
Remove the columns that you're no longer renaming ("pointing" and "sca") from this translation dict.